PREREQUISITES.md

```diff
@@ -31,11 +31,35 @@ WP-CLI version:	1.4.1
 
 Update, if needed, to the latest stable release with `wp cli update`.
 
+#### Setting defaults for the wp valet command
+As with other `wp-cli` commands, you can set default attributes when running `wp valet`.
+
+Simply add the appropriate details in `~/.wp-cli/config.yml`:
+
+```
+valet new:
+	project: wp
+	# in: override - defaults to current directory
+    version: latest
+    # locale: - use if not English
+    db: mysql
+    # dbname: defaults to wp_name
+    dbuser: root # or any other local user capable of creating databases (MySQL only)
+    dbpass: # enter the appropriate password if necessary (MySQL only)
+    dbprefix: wp_
+    admin_user: admin
+    admin_pass: admin
+    # unsecure - use without colon if you'd like to override HTTPS
+    # portable - see above
```

Comment:
Boolean options such as `--unsecure` can be set in the config as `unsecure: true`